### PR TITLE
feature(importer-ergonode): Create ergonode import based on CSV files

### PR DESCRIPTION
**Pull request by piotrkreft**
# Description

Issue # 
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ ] Unit Tests
- [x] Behat Test

# Checklist:
- [ ] I have read the contribution requirements and fulfill them.
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -56,6 +57,7 @@
         "symfony/translation": "^4.4",
         "symfony/twig-bundle": "^4.4",
         "symfony/validator": "^4.4",
+        "symfony/var-dumper": "^5.1",
```
Comment (wfajczyk) [inline composer.json]: Change to `require-dev`.

File: module/designer/src/Domain/ValueObject/TemplateElement/AttributeTemplateElementProperty.php [hunk 2]
```diff
@@ -53,4 +53,12 @@ public function isRequired(): bool
     {
         return $this->required;
     }
+
+    public function isEqual(object $property): bool
```
Comment (wfajczyk) [inline module/designer/src/Domain/ValueObject/TemplateElement/AttributeTemplateElementProperty.php:57]:
```suggestion
    public function isEqual(AttributeTemplateElementProperty $property): bool
```

File: module/designer/src/Domain/ValueObject/TemplateElement/UiTemplateElementProperty.php [hunk 3]
```diff
@@ -40,4 +40,11 @@ public function getLabel(): string
     {
         return $this->label;
     }
+
+    public function isEqual(object $property): bool
```
Comment (wfajczyk) [inline module/designer/src/Domain/ValueObject/TemplateElement/UiTemplateElementProperty.php:44]:
```suggestion
    public function isEqual(UiTemplateElementProperty $property): bool
```

File: module/designer/src/Domain/ValueObject/TemplateElementPropertyInterface.php [hunk 4]
```diff
@@ -12,4 +12,5 @@
 interface TemplateElementPropertyInterface
 {
     public function getVariant(): string;
+    public function isEqual(object $property): bool;
```
Comment (wfajczyk) [inline module/designer/src/Domain/ValueObject/TemplateElementPropertyInterface.php:15]:
```suggestion
    public function isEqual(TemplateElementPropertyInterface $property): bool;
```

File: module/exporter-file/src/Infrastructure/Processor/ProductProcessor.php [hunk 5]
```diff
@@ -64,17 +75,18 @@ public function process(FileExportChannel $channel, AbstractProduct $product): E
...
+        array $attributes,
+        Template $template
+    ): LanguageData {
         $result = new LanguageData();
         $result->set('_id', $product->getId()->getValue());
         $result->set('_sku', $product->getSku()->getValue());
         $result->set('_type', $product->getType());
         $result->set('_language', $language->getCode());
-        $result->set('_template', $product->getTemplateId()->getValue());
+        $result->set('_template', $template->getName());
```
Comment (wfajczyk) [inline module/exporter-file/src/Infrastructure/Processor/ProductProcessor.php:89]: Why change from `id` to `name`?

File: module/importer-ergonode-1/composer.json [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    "sort-packages": true
+  },
+  "require": {
+    "php": "^7.4",
+    "ext-pdo": "*",
+    "ext-zip": "*",
+    "jms/serializer": "^3.4",
+    "jms/serializer-bundle": "^3.4",
+    "ergonode/importer": "^0.10.0",
```
Comment (wfajczyk) [inline module/importer-ergonode-1/composer.json:20]:
```suggestion
    "ergonode/importer": "^1.0.0-beta.6"
```

**Review by wfajczyk — CHANGES_REQUESTED** [review]

File: module/exporter-file/src/Infrastructure/Processor/ProductProcessor.php (hunk 5 above)
Comment (piotrkreft) [reply module/exporter-file/src/Infrastructure/Processor/ProductProcessor.php:89]: Basically via import new ids being created therefore those cant really be used

File: module/designer/src/Domain/ValueObject/TemplateElement/UiTemplateElementProperty.php (hunk 3 above)
Comment (piotrkreft) [reply module/designer/src/Domain/ValueObject/TemplateElement/UiTemplateElementProperty.php]: inherited - limited by interface

File: module/designer/src/Domain/ValueObject/TemplateElement/AttributeTemplateElementProperty.php (hunk 2 above)
Comment (piotrkreft) [reply module/designer/src/Domain/ValueObject/TemplateElement/AttributeTemplateElementProperty.php]: inherited - limited by interface

**Review by wfajczyk — APPROVED** [review]

**Review by wiewiurdp — APPROVED** [review]

**Pull request merged**